tests/bsim/bluetooth/host/misc/disable/src/main_disable.c

```diff
@@ -35,6 +35,65 @@ static void test_disable_main(void)
 	PASS("Disable test passed\n");
 }
 
+static void test_disable_set_default_id(void)
+{
+	for (int i = 0; i < NUM_ITERATIONS; i++) {
+
+		/* FIXME: Temporary workaround to get around a bug in the controller
+		* Open PR: https://github.com/zephyrproject-rtos/zephyr/pull/73342
+		*/
+		k_sleep(K_MSEC(1));
+
+		int err;
+		size_t id_count;
+		bt_addr_le_t stored_id;
+		bt_addr_le_t addr = {.type = BT_ADDR_LE_RANDOM,
+				     .a = {.val = {i, 2, 3, 4, 5, 0xc0}}};
+
+		err = bt_id_create(&addr, NULL);
+		if (err != 0) {
+			FAIL("Creating ID failed (err %d)\n", err);
+		}
+
+		err = bt_enable(NULL);
+		if (err != 0) {
+			FAIL("Enable failed (err %d)\n", err);
+		}
+
+		bt_id_get(NULL, &id_count);
+		if (id_count != 1) {
+			FAIL("Expected only one ID, but got: %u\n", id_count);
+		}
+
+		id_count = 1;
+		bt_id_get(&stored_id, &id_count);
+		if (id_count != 1) {
+			FAIL("Expected only one ID, but got: %u\n", id_count);
+		}
+
+		if (!bt_addr_le_eq(&stored_id, &addr)) {
+			uint8_t addr_set_str[BT_ADDR_LE_STR_LEN];
+			uint8_t addr_stored_str[BT_ADDR_LE_STR_LEN];
+
+			bt_addr_le_to_str(&addr, addr_set_str, BT_ADDR_LE_STR_LEN);
+			bt_addr_le_to_str(&stored_id, addr_stored_str, BT_ADDR_LE_STR_LEN);
+			FAIL("Expected stored ID to be equal to set ID: %s, %s\n",
+			     addr_set_str, addr_stored_str);
+		}
+
+		err = bt_disable();
+		if (err) {
+			FAIL("Enable failed (err %d)\n", err);
+		}
+	}
+
+	/* BSIM test framework requires at to run some time before marking a
+	 * test case as passed. */
+	k_sleep(K_MSEC(1));
```

Comment:
@aescolar , are you aware of this issue? If I create a test case only containing the line `PASS("")` the test fails because it thinks it is still running

Reply:
Is it because the other device(s) are still initializing and did not get to pass?

Reply:
Created https://github.com/zephyrproject-rtos/zephyr/pull/73348